manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64)

```

```suggestion
PackageIdentifier: Microsoft.DotNet.AspNetCore.2_1
```
Please update file name and path to match this change.